content/en/_index.Rmd

```diff
@@ -341,6 +341,15 @@ day_new_f <- which(as.character(ott_observed_episodes[["date"]]) == ymd("2020-05
 f_seg[seq(day_new_f, length(f_seg))] <- 2
 day_ch <- which(as.character(ott_observed_episodes[["date"]]) == ymd("2020-06-12")) # Date of phase 2 reopening
 f_seg[seq(day_ch, length(f_seg))] <- 3
+
+day_ch <- which(as.character(ott_observed_episodes[["date"]]) == ymd("2020-07-17")) # Date of phase 3 reopening
```

Comment:
Addition of phase 3 reopening, modified phase 2, and return to phase 3 contact rate breakpoints.